Bangazon/Models/PaymentType.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+//Written by: Eliza Meeks
+
+namespace BangazonCLI
+{
+    // Gets and sets public properties of a payment type.
+    public class PaymentType
+    {
+        public int paymentTypeID {get;}
+        public int customerID {get; set;}
+        public int accountNumber {get; set;}
+        public string name {get; set;}
+        public Customer customer {get; set;}
```

Comment:
Is this necessary? I can't think of an instance in which we'd need to call on the Customer FROM the payment type. Usually we'd take the customer ID, get all their payment types from SQL, then do whatever we need with the payment type (like add it to an order).